src/main/java/com/wcc/platform/service/ProgrammeService.java

```diff
@@ -27,14 +28,17 @@ public ProgrammeService(final ObjectMapper objectMapper) {
    * @return Programme Page json response
    */
   public ProgrammePage getProgramme(final ProgramType programType) {
-    try {
-      String data = null;
-      if (ProgramType.BOOK_CLUB.equals(programType)) {
-        data = FileUtil.readFileAsString(ApiResourcesFile.PROG_BOOK_CLUB.getFileName());
+    if (ProgramType.BOOK_CLUB.equals(programType)) {
```

Comment:
Alternative approach -- Define a map of ProgramType and file name. And then loop over the map. This make it extendable for more ProgramType in future. We need to just update the map with more programtype to filename mapping.

Reply:
We are going to take care about this in another PR